lib/common/database.dart

```diff
@@ -173,41 +192,23 @@ class DatabaseService {
     return getListOfEventsInDateRange(dateStart: date, dateEnd: tomorrow);
   }
 
-  Future<void> addEvent(String eventID, Event event) async {
-    var doc = await users.doc(userid).collection("events").doc(eventID).get();
-    // can't add an event with the same name
-    if (doc.exists) {
-      throw Future.error("Event ID already exists!");
-    }
-    return await users
-        .doc(userid)
-        .collection("events")
-        .doc(eventID)
-        .set(event.toMap());
-  }
-
-  /// Add/set the user event. The eventID is necessary.
-  ///
-  /// Every possible option to set is an argument
-  /// required: String eventID, String eventName, Set<String> eventTags, num timeStart, num timeEnd
-  /// optional: String eventDescription,, String, eventLocation, String eventColor, bool recurrenceEnabled, num recurrenceTimeStart, num recurrenceTimeEnd, List<bool> recurrenceDates
-  Future<void> addEventArgs(
-      {required String eventID, required Event event}) async {
-    return await addEvent(eventID, event);
-  }
-
   /// Change an option in the event
   ///
   /// needs a event ID and a map of the new option
   /// map ex: {"optionName": "optionValue"}
   Future<void> updateEventOption(
       String eventID, Map<String, dynamic> newOptions) async {
-    return users.doc(userid).collection("events").doc(eventID).update(newOptions);
+    return users
+        .doc(userid)
+        .collection("events")
+        .doc(eventID)
+        .update(newOptions);
   }
 
-  Future<void> updateEventName(String oldEventID, String newEventID) async {
+  Future<void> updateEventId(String oldEventID, String newEventID) async {
```

Comment:
can we remove this? not sure how it would be used

Reply:
yeah, it used to be there as one of the old setters/getters, so it's no longer necessary